### PR TITLE
Kevin autotest

### PR DESCRIPTION
**Pull request by weizhang-2706**
## Describe your changes

- [ ] Bug fix
- [ ] New feature

Explain what you did:
> ...

## Issue ticket (add number)

> https://github.com/FRC2706/2024-2706-Robot-Code/issues/...

## Checklist before requesting a review
- [ ] I have performed a self-review of my code.
- [ ] I have tested thoroughly.
- [ ] I have written documentation and explained about my changes to the best of my ability.

### CODE REVIEW

File: src/main/java/frc/robot/commands/auto/AutoRoutines.java [hunk 1]
```diff
@@ -5,31 +5,47 @@
...
 import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.IntakeControl;
+import frc.robot.commands.MakeIntakeMotorSpin;
+import frc.robot.commands.Shooter_tuner;
+import frc.robot.subsystems.IntakeSubsystem;
 import frc.robot.subsystems.SwerveSubsystem;
 
 public class AutoRoutines extends SubsystemBase {
-    PathPlannerAuto four_note = new PathPlannerAuto("4note");
+    PathPlannerPath path1 = PathPlannerPath.fromPathFile("4 note");
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/commands/auto/AutoRoutines.java]: Need clean up.
1. Under paths: remove 4 note/ben /Pick Up Note/Place Note/feeder right starting/2mStraightLines /2mStraightLines/MakeIntakeMotorSpin
2. Under autos: remove 4note/MakeIntakeMotorSpin Auto Test/One Note/Parallel Auto Test/Sequenctial Auto Test/
3. Clean up codes here accordingly.

File: src/main/java/frc/robot/commands/auto/AutoRoutines.java [hunk 2]
```diff
@@ -5,31 +5,47 @@
...
+    // public static AutoRoutines getInstance(){
+    //     if(instance == null){
+    //         instance = new AutoRoutines();
+    //     }
+    //     return instance;
+    // }
 
     public static void registerCommandsToPathplanner() {
-        
+        IntakeSubsystem.getInstance().setDefaultCommand(IntakeSubsystem.getInstance().autoIntake());
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/commands/auto/AutoRoutines.java]: Remove this line. This has been set in NewRobotContainer.

File: src/main/java/frc/robot/subsystems/SwerveModule.java [hunk 3]
```diff
@@ -165,10 +165,10 @@ private void configDriveMotor() {
...
     configureSpark("Drive velocity conversion factor", () -> driveEncoder.setVelocityConversionFactor(Config.Swerve.driveConversionVelocityFactor));
     configureSpark("Drive position conversion factor", () -> driveEncoder.setPositionConversionFactor(Config.Swerve.driveConversionPositionFactor));
-    configureSpark("Drive set P", () -> driveController.setP(Config.Swerve.angleKP));
-    configureSpark("Drive set I", () -> driveController.setI(Config.Swerve.angleKI));
-    configureSpark("Drive set D", () -> driveController.setD(Config.Swerve.angleKD));
-    configureSpark("Drive set FF", () -> driveController.setFF(Config.Swerve.angleKFF));
+    configureSpark("Drive set P", () -> driveController.setP(Config.Swerve.driveKP));
+    configureSpark("Drive set I", () -> driveController.setI(Config.Swerve.driveKI));
+    configureSpark("Drive set D", () -> driveController.setD(Config.Swerve.driveKD));
+    configureSpark("Drive set FF", () -> driveController.setFF(Config.Swerve.driveKFF));
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/subsystems/SwerveModule.java:171]: These are important bug fixes.

File: src/main/java/frc/robot/robotcontainers/NewRobotContainer.java [hunk 4]
```diff
@@ -67,10 +44,10 @@ public class NewRobotContainer extends RobotContainer {
   private TunableNumber shooterDesiredVoltage = new TunableNumber("Shooter/desired Voltage", 0);
     
   String tableName = "SwerveChassis";
-  private NetworkTable swerveTable = NetworkTableInstance.getDefault().getTable(tableName);
-  private IntegerEntry entryAutoRoutine;
+  //private NetworkTable swerveTable = NetworkTableInstance.getDefault().getTable(tableName);
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/robotcontainers/NewRobotContainer.java]: Need clean up this file.

**Review by Khalil-Burns — APPROVED** [review]

**Review by Khalil-Burns — APPROVED** [review]

**Review by ErikCald — APPROVED** [review]

**Pull request merged**